create-presigned-download-url/src/main/java/no/unit/nva/download/publication/file/PresignedUri.java

```diff
@@ -2,11 +2,13 @@
 
 import com.fasterxml.jackson.annotation.JsonGetter;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 
 import java.net.URI;
 import java.time.Instant;
 import java.util.Date;
+import nva.commons.core.JacocoGenerated;
 
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 public class PresignedUri {
```

Comment:
Nothing you have done, but either annotate constructor with @JsonCreator and annotate parameters with @JsonProperty and make the DTO immutable, or annotate field with @JsonProperty.

Reply:
Delvis fixet, gikk for default constructor pluss custom constructor og JsonProperty med setters. Litt stygt, men vi trenger JsonProperty annotering for å få med context feltet på vei ut.

Reply:
Fikk fikset at den tar inn en Date i konstruktøren og getteren returnerte en instant.